targets/TARGET_ONSEMI/TARGET_NCS36510/PinNames.h

```diff
@@ -128,8 +128,15 @@ typedef enum {
     LED_YELLOW  = GPIO6,
     LED_BLUE    = GPIO5,
     USER_BUTTON = GPIO7,  /*NEW connection on NCS36510-RF Rev 1.1 - Alias of SW1 */
+
     SW1 = GPIO7,  /*NEW connection on NCS36510-RF Rev 1.1 */
     SW2 = GPIO10,   /*NEW connection on NCS36510-RF Rev 1.1 */
+
+    // Standardized button names
+    BUTTON1 = USER_BUTTON,
+    BUTTON2 = SW1,
```

Comment:
Still not quite right, `USER_BUTTON` and `SW1` both map to `GPIO7`. I would think the correct definitions would be as follows:

```
BUTTON1 = SW1,
BUTTON2 = SW2,
```